app/utils/errors.js

```diff
@@ -0,0 +1,13 @@
+export class WebIntegrationError extends Error {
+  constructor(...args) {
+    super(...args);
+    Error.captureStackTrace(this, WebIntegrationError);
+  }
+}
+
+export class OtherError extends Error {
```

Comment:
`OtherError` is not a descriptive name. `ApplicationError`, for example, would be a better name custom generic error.